src/seedu/addressbook/AddressBook.java

```diff
@@ -188,12 +188,19 @@
      * This is a subset of the full list. Deleting persons in the pull list does not delete
      * those persons from this list.
      */
-    private static ArrayList<String[]> latestPersonListingView = getAllPersonsInAddressBook(); // initial view is of all
+    private static ArrayList<String[]> _latestPersonListingView = getAllPersonsInAddressBook(); // initial view is of all
 
     /**
+     * Added constants to indicate the number of constants passed.
+     */
+    private static final int ARGC_INVALID = 2;
+    private static final int ARGC_GIVEN_FILE = 1;
+    private static final int ARGC_DEFAULT_FILE = 0;
+    
+    /**
```

Comment:
You handled the magic numbers.  But why use argc?  Typically, argc means number of arguments.  You can make it even more meaningful.